.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+<!--
+Thanks for submitting a pull request!
+
+If you are adding or updating translations, here's a helper guide: https://fabulously-optimized.gitbook.io/modpack/readme/language-support
+If this is a translation update or addition for Mod Menu Helper, make sure it will fit in Minecraft either by testing it using this modpack and/or this online tool: https://fabulously-optimized.github.io/Mod-Menu-Helper-Size-Checker/
+
+Keep in mind that we **do not** accept mod or configuration changes using pull requests, please create an issue here instead: https://github.com/Fabulously-Optimized/fabulously-optimized/issues/new/choose.
```

Comment:
Use `https://github.com/Fabulously-Optimized/fabulously-optimized/issues/new/choose/` as github for some reason will assume the period belongs inside the URL which isn't what we want.

Reply:
in vscode this didn't count as part of the url

Reply:
~~forgot to resolve this :ioa:~~